sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -35,11 +36,19 @@ export class AzureSASCredential implements SASCredential {
     update(newSignature: string): void;
 }
 
+// @public
+export function computeTokenType(accessToken: AccessToken): string;
```

Comment:
Does this need to be public?

Reply:
Yes to be used in Identity when needed

Reply:
Jeff had suggested these helper APIs so that we can make tokenType field optional

Reply:
Do any other packages beside identity use this? Is it possible to have this in identity for now?

Reply:
To chip in my 2c -- I don't think the helper is needed and it is probably enough to just document the default value of the property. Additional public API surface for what would otherwise be something as simple as `const tokenType = accessToken.tokenType ?? "Bearer";` seems unnecessary. +1 for just implementing this in Identity for now; if it becomes a pain point we can always revisit and add the helper here in future.

Reply:
We need the computeTokenType for the authentication headers

Reply:
> Do any other packages beside identity use this? Is it possible to have this in identity for now?
The core-rest-pipeline will also need it in the future

Reply:
We need to keep the getTokenType in core bcz it will be used to form Authentication Headers

Reply:
The context was also that all the other languages have tokenType as a required field. For JS, having it required would be a breaking change, so we decided to have a helper function instead.
@xirzec